modules/rosa-install-uninstall-addon.adoc

```diff
@@ -0,0 +1,108 @@
+:system-module-type: REFERENCE
+// Module included in the following assemblies:
```

Comment:
@jonquilwilliams please add rosa describe addon to this section.

Reply:
@jeremyeder  This PR is just for the migration to the new repo. The actual documentation card for CLI is: https://issues.redhat.com/browse/OSDOCS-1120

And I did the addon commands yesterday. You can view it in this PR: https://github.com/openshift/managed-openshift-docs/pull/9